### ISSUE TITLE
Question about the Taxi dataset

### ISSUE THREAD

**Issue by won-bae** [data-iitd/imtpp#1]
Hi authors, thank you for sharing the code for the awesome work.
While I was trying to reproduce some of the results, I ran into a problem with real datasets, especially, NYC taxi 2013 dataset. In the paper, you mentioned "Each sequence corresponds to a series of time-stamped pick-up and drop-off events of a taxi in New York City with locations as marks." for NYC taxi. I assume you used pick-up time as inputs. 

But then, how did you separate the dataset into sequences? You also mentioned that "Given a stream of N observed events $S_N$, we split them into training $S_K$ and test set $S_N \backslash S_K$, where the training set (test set) consists of first 80% (last 20%) events, i.e., $K = [0:8N]$." But it did not explain how the sequences were generated. 

Also, according to my analysis on the dataset, there are many overlapping events which are not allowed for TPP. How did you handle those overlapping events?

Thank you so much in advance!

**Comment by gvinayak** [comment]
Hi @won-bae ,
Thanks a lot for your interest in our work.

The taxi dataset contains the details of pick-up as well as drop-offs for each taxi trip. Specifically, for every taxi in NYC, we have a time-stamped sequence of pick-up and drop-off events with different locations having different event types. The dataset that we have used is available here: https://drive.google.com/drive/folders/1tZ6xODd3tO3qgSp2gp-jqYOdfh3aj827

I'm not able to understand what you meant by overlapping events. If you were referring to overlapping 'intervals' due to the train-test split, then we ignore that. This is a standard technique followed in recommender systems, and is more suitable for evaluating the performance across the datasets derived from Amazon. However, there are no overlapping 'events' between the train and test sets. I request you to please check the dataset available in the above link.

Thanks again for your interest!
